### PR TITLE
Bring AWS Organizations accounts into management-account management (Analytical Platform to HMPPS)

### PR DESCRIPTION
**Pull request by jakemulley**

### CODE REVIEW

File: management-account/terraform/organizations-accounts-analytical-platform.tf [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      name,
+      role_name,
+    ]
+  }
+}
+
+# Analytical Platform Data Engineering Sandbox
+resource "aws_organizations_account" "analytical_platform_data_engineering_sandbox" {
+  name                       = "Analytical Platform Data Engineering Sandbox"
+  email                      = replace(local.aws_account_email_addresses_template, "{email}", "analytics-platform-dev")
```
Comment (ben-al) [inline management-account/terraform/organizations-accounts-analytical-platform.tf:31]: It's Sandbox so could be dev email on purpose, just leaving a note incase.

Comment (jakemulley) [reply management-account/terraform/organizations-accounts-analytical-platform.tf:31]: Cheers for checking! This particular account has had many reincarnations, and this is the email that is currently set - it's on my todo to change it to something more meaningful (but required logging in as the AWS root IAM user for that account) 👍🏼 .

File: management-account/terraform/organizations-accounts-hmpps-electronic-monitoring-case-management.tf [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+resource "aws_organizations_account" "electronic_monitoring_case_management_prod" {
+  name                       = "Electronic Monitoring Case Management prod"
+  email                      = replace(local.aws_account_email_addresses_template, "{email}", "hmpps-em-cm-prod")
+  iam_user_access_to_billing = "ALLOW"
+  parent_id                  = aws_organizations_organizational_unit.hmpps_electronic_monitoring_case_management.id
+
+  tags = merge(local.tags_electronic_monitoring_case_management, {
+
```
Comment (ben-al) [inline management-account/terraform/organizations-accounts-hmpps-electronic-monitoring-case-management.tf:52]: Same comment on `is-production`

**Review by ben-al — APPROVED** [review]
Left a few comments that are probably nothing but LGTM

File: management-account/terraform/organizations-accounts-hmpps-electronic-monitoring.tf [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+      name,
+      role_name,
+    ]
+  }
+}
+
+# Electronic Monitoring Protective Monitoring
+resource "aws_organizations_account" "electronic_monitoring_protective_monitoring" {
+  name                       = "Electronic Monitoring Protective Monitoring"
+  email                      = replace(local.aws_account_email_addresses_template, "{email}", "hmpps-em-promon")
```
[the remark this answers is not in the document]
Comment (jakemulley) [reply management-account/terraform/organizations-accounts-hmpps-electronic-monitoring.tf:155]: Good spot, but alas, it is correct.

File: management-account/terraform/organizations-accounts-hmpps-electronic-monitoring.tf [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+      name,
+      role_name,
+    ]
+  }
+}
+
+# Electronic Monitoring Shared Networking (non-prod)
+resource "aws_organizations_account" "electronic_monitoring_shared_networking_non_prod" {
+  name                       = "Electronic Monitoring Shared Networking (non-prod)"
+  email                      = replace(local.aws_account_email_addresses_template, "{email}", "hmpps-em-shared-networking-nonprod")
```
[the remark this answers is not in the document]
Comment (jakemulley) [reply management-account/terraform/organizations-accounts-hmpps-electronic-monitoring.tf:218]: It's correct (current email address for that account).

**Pull request merged**